coldfront/core/user/migrations/0008_auto_20210426_1353.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.1.7 on 2021-04-26 13:53
```

Comment:
Considering the changes in `0007` get cancelled here, we don't need two migrations.

The following should work:
1. Un-migrate these two using: `python manage.py migrate user 0006`.
2. Delete these two migration files.
3. Run `makemigrations` again.